Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.8.6-alpine
+
+## Prepare docker base dependency
+RUN set -euxo pipefail; \
+    apk update; \
+    apk add --no-cache \
+        tini \
+        su-exec
+
+## Create user with home = work directory
+RUN set -euxo pipefail; \
+    addgroup --gid 1001 free-stuff-matrixbot; \
+    adduser --home '/opt/free-stuff-matrixbot' --gecos '' --ingroup free-stuff-matrixbot --disabled-password --uid 1001 free-stuff-matrixbot
+WORKDIR /opt/free-stuff-matrixbot
+
+## Add entrypoint
+COPY ["docker-entrypoint.sh", "/usr/local/bin/docker-entrypoint.sh"]
+ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/docker-entrypoint.sh"]
+
+## Install system dependencies for python dependencies
+RUN set -euxo pipefail; \
+    apk update; \
+    apk add --no-cache \
+        build-base \
+        gcc
+
+## Install dependencies
+COPY requirements.txt .
+RUN set -euxo pipefail; \
```

Comment:
See above, especially for a single command

Reply:
OK, single command is a thing, but on the other hand I want `u` and `x` and then it's two command because of `set`. And to keep it simple I started simply putting the full `set -euxo pipefail` everywhere :man_shrugging: